src/components/add-movie-form/add-movie-form.tsx

```diff
@@ -0,0 +1,113 @@
+import { Button } from 'components/button';
+import { Input } from 'components/input';
+import { ReleaseDatePicker } from 'components/release-date-picker';
+import { Select } from 'components/select';
+import React, { FC, FormEvent, useState } from 'react';
+import {
+  AddMovieButtonContainer,
+  AddMovieFormContainer,
+  AddMovieFormInner,
+  AddMovieFormTitle,
+  AddMovieFormWrapper,
+  CloseButton,
+} from './styles';
+
+const initialValues = {
+  genre: '',
+  movie: '',
+  overview: '',
+  rating: '',
+  runtime: '',
+  title: '',
+};
+
+const selectOptions = [
+  { id: 1, name: 'Crime' },
+  { id: 2, name: 'Documentary' },
+  { id: 3, name: 'Horror' },
+  { id: 4, name: 'Comedy' },
+];
+
+export const AddMovieForm: FC = () => {
+  const [values, setValues] = useState(initialValues);
+
+  const handleOnChange = ({ target }): void => {
+    const { name, value } = target;
+
+    setValues({
+      ...values,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    console.log(JSON.stringify(values, null, 2));
```

Comment:
I would suggest to remove console log after you tested tour code with it

Reply:
yep, sure